lib/maintainers.nix

```diff
@@ -100,4 +100,10 @@
     githubId = 6452260;
     name = "Andrew Plaza";
   };
+  ZainKergaye = {
+    email = "zain@zkergaye.me";
+    github = "zainkergaye";
+    githubId = 62440012;
+    name = "Zain Kergaye";
+  };
```

Comment:
This should be done in a separate commit, titled `maintainers: add ZainKergaye`

Reply:
This isn't resolved, but the commit history needs squashing anyway.

Once the PR is done, could you squash so that we only have two commits:
- `maintainers: add ZainKergaye`
- `plugins/wrapping: init`

Reply:
How do I go about changing the commits?

Reply:
There's a few ways you can do it, e.g. with an "interactive" rebase. But in this case I would do:
```
# reset to the base commit without modifying the working tree
git reset --mixed cb2b76c1a9ec067ed0c449080f4973fecf8532ef

git add lib/maintainers.nix
git commit -m "maintainers: add ZainKergaye"

git add plugins
git add tests
git commit -m "plugins/wrapping: init"

git push --force-with-lease
```